netsim/ansible/templates/initial/sros.j2

```diff
@@ -21,7 +22,7 @@
 {%- endmacro %}
 
 {% macro declare_interface(i,name=None) %}
```

Comment:
Not sure if the `name` parameter is really used anywhere, could probably remove it and use `i.ifname` instead